.ci/pitest.sh

```diff
@@ -131,10 +131,11 @@ pitest-ant)
 pitest-indentation)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=(
+  "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>                &#38;&#38; tree.getColumnNo() &#60; realStart.getColumnNo()) {</span></pre></td></tr>"
   "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>            if (colNum == null || thisLineColumn &#60; colNum) {</span></pre></td></tr>"
-  "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>        if (currLine &#60; realStart) {</span></pre></td></tr>"
   "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>            if (toTest.getColumnNo() &#60; first.getColumnNo()) {</span></pre></td></tr>"
   "ArrayInitHandler.java.html:<td class='covered'><pre><span  class='survived'>        if (firstChildPos &#62;= 0) {</span></pre></td></tr>"
+  "ArrayInitHandler.java.html:<td class='covered'><pre><span  class='survived'>        return true;</span></pre></td></tr>"
```

Comment:
please expain this mutation

Reply:
I am working on removal of it, and stabilizing all the diff report. My next commit will not have this pitest mutation :)

Reply:
Done